.github/workflows/cypress-workflow-vanilla-source-based.yml

```diff
@@ -88,8 +88,8 @@ jobs:
         uses: cypress-io/github-action@v2
         with:
           working-directory: monetery-test
-          command: yarn run cypress run --browser chrome --headless --spec 'cypress/integration/vanilla-opensearch-dashboards/*.js'
-          wait-on: 'http://localhost:5601'
+          command: yarn run cypress run --browser chrome --headless --env SECURITY_ENABLED=true --spec 'cypress/integration/core-opensearch-dashboards/vanilla-opensearch-dashboards/*.js'
```

Comment:
There is no plugin for this flow. Shall this be `false` instead? Interesting that specifying `true` didn't fail the test.

Reply:
Yes, should be false - I'll update here as well.

Reply:
I think it still succeeds since the only difference is the requests having the added `admin/admin` auth, which is probably just ignored if no security plugin exists.